spec/features/02_user_can_see_a_cryptid_show_page_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+feature 'user can see a cryptid show page', %Q{
+  As a user
+  I want to see a single cryptid
+  To find out more about it
+} do
+  #Acceptance Criteria
+  # * If I go to the cryptid show page I see info about the cryptid
+  let!(:test_user) { FactoryBot.create(:user) }
+  let!(:cryptid) do
+     Cryptid.create!(
+      name: "Dragon",
+      description: "Breathes fire",
+      user: test_user)
```

Comment:
This looks great! I'd just push the closing parentheses down to the next line in your next PR.